docs/3.tutorials/crosswords/03-intermediate/03-linkdrop.md

```diff
@@ -26,10 +26,12 @@ Let's discuss how this testnet account gets created.
 Notice the new account will end in `.testnet`. This is because the account `testnet` will create a subaccount (like we learned about [earlier in this tutorial](../01-basics/02-add-functions-call.md#create-a-subaccount)) called `vacant-name.testnet`.
 
 There are two ways to create this subaccount:
+
 1. Use a full-access key for the account `testnet` to sign a transaction with the `CreateAccount` Action.
-2. Have a smart contract deployed to the `testnet` account that has a Promise executing the `CreateAccount` Action. (More info about writing a [`CreateAccount` Promise](/sdk/rust/promises/create-account).)
+2. In a smart contract deployed to the `testnet` account, call a Promise executing the `CreateAccount` Action. (More info about writing a [`CreateAccount` Promise](/sdk/rust/promises/create-account).)
+
+(In the example below that uses NEAR CLI to create a new account, it's calling the Promise on the linkdrop contract that is deployed to the top level "near" account on mainnet.)
```

Comment:
I wouldn't say the promise is being called. The function is called, which is async, which creates a promise. But in general one calls functions that execute logic. If this is performed asynchronously then promises are created